onnxruntime/core/platform/posix/env.cc

```diff
@@ -359,6 +359,10 @@ class PosixEnv : public Env {
     return S_ISDIR(sb.st_mode);
   }
 
+  std::string GetRuntimePath() const override {
+    return "./";
```

Comment:
This doesn't seem correct based on the Windows implementation.

Reply:
I think the shared library loading strategy is different between posix and windows. previously it is implement by empty... but i am not an expert on that. @RyanUnderhill